packages/editor/src/dataviews/types.ts

```diff
@@ -11,11 +11,16 @@ export interface BasePost {
 	status?: PostStatus;
 	title: string | { rendered: string };
 	type: string;
+	id: string | number;
 }
 export interface TemplateOrTemplatePart extends BasePost {
-	type: 'template' | 'template-part';
+	type: 'wp_template' | 'wp_template_part';
```

Comment:
So, this was a TS bug? I'm curious why in our code base it didn't have any errors? 🤔

Reply:
yes, but since these types are very new, they didn't impact much.